jwst/wfs_combine/wfs_combine.py

```diff
@@ -322,10 +323,15 @@ def create_combined(self, im_1_a, im_2_a):
         err_data_1 = im_1_a.err.copy()
         err_data_2 = im_2_a.err.copy()
 
-        wh_1_bad_2_good = (dq_data_1 != 0) & (dq_data_2 == 0)
-        wh_1_bad_2_bad = (dq_data_1 != 0) & (dq_data_2 != 0)
-        wh_1_good_2_good = (dq_data_1 == 0) & (dq_data_2 == 0)
-        wh_1_good_2_bad = (dq_data_1 == 0) & (dq_data_2 != 0)
+        bad1 = np.bitwise_and(dq_data_1, dqflags.pixel['DO_NOT_USE'])
```

Comment:
`np.bitwise_and` is returning an array of 0 and 1, so a bitmask.  This is then used to index another array as if it were a boolean array below, which it is not.  This needs to be converted to a boolean if it is to be used for indexing, otherwise it is just changing indexes 0 and 1 multiple times.

Same for 2 lines down.